docs/configuration.md

```diff
@@ -0,0 +1,124 @@
+# Configuration settings
+
+This document provides an overview of configuration settings for increasing hub performance.
+
```

Comment:
nit: it might be worth having a short table of contents at the top here for at least the top-level sections (the ## sections); the more I dig into markdown github docs the more I want ToCs on them if we're not going to use mkdocs or something (though we could use mkdocs which gives us a nav sidebar), or maybe github pages (I haven't used those yet).

Reply:
I'll get that added.